gatsby/transformerplugins.rst

```diff
@@ -57,20 +58,25 @@ And add it into our `gatsby-config.js` file in the plugins list:
   ]
   ...
 
-This plugins reads all the files generated by the `gatsby-source-filesystem` plugin, and extract data from the items that he can handle (Markdown files for example),
-generating new nodes in our GraphQL model with more useful metadata.
+This plugin reads all the files generated by the `gatsby-source-filesystem` plugin and extract data from the items that he can handle (Markdown files for example), generating new nodes in GraphQL model with more useful metadata.
+
+If we restart our server and see GraphiQL queries we will find two new available queries: `allMarkdownRemark` and `markDownRemark`.
 
-Let's try to restart our server and see what's new in GraphiQL queries: there are two new available queries: `allMarkdownRemark` and `markDownRemark`.
+Every source or transform plugin creates a set of new queries and `nodes`.
+
+A node is a data entity generated by that plugin that exposes several attributes, and get its name from its `type` attribute.
+
+In this case, the remark transofermer creates a list of nodes called `markDownRemark`.
+
+GatsbyJS automatically generates also a GraphQL query type to get all the nodes of a certain type (an filter them by their attributes values) with a standard name: "all" + the node type name (in our case: allMarkdownRemark).
 
 Exercise
 ++++++++
 
-We want to create a blog, so we need to edit our `pages/index.js` to list all blogposts.
-
-You need to provide these informations:
+We want to create a blog, so we need to edit our `pages/index.js` to list all blogposts with these informations:
```

Comment:
Double backticks needed.

Either:

```
``blogposts``
```
or "blogposts", depending on context.

s/these informations/the following information